charts/cass-operator-chart/templates/deployment.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Values.deploymentName }}
+  namespace: {{ .Values.cassOperatorNamespace }}
+spec:
+  replicas: {{ .Values.deploymentReplicas }}
+  selector:
+    matchLabels:
+      name: cass-operator
+  template:
+    metadata:
+      labels:
+        name: cass-operator
+    spec:
+      serviceAccountName: {{ .Values.serviceAccountName }}
+      volumes:
+      - name: cass-operator-certs-volume
+        secret:
+          secretName: cass-operator-webhook-config
+      containers:
+      - name: cass-operator
+        image: {{ .Values.cassOperatorImage }}
+        imagePullPolicy: {{ .Values.cassOperatorImagePullPolicy }}
```

Comment:
Is the `cassOperator` prefix needed in this case?

Reply:
Yeah, it's kinda wonky either way because the operator image is "cass-operator".  I'll go ahead and remove the prefix here.